### ISSUE TITLE
Create an account just for token creation

### ISSUE THREAD

**Issue by maelle** [ropensci-review-tools/pkgcheck#140]
as mentioned by @assignUser https://github.com/ropensci-review-tools/pkgcheck/issues/123#issue-1109573181

Advantages

- no access to other repos
- if using an address like `accounts+reviewtools@ropensci.org` and well named tokens, several people would be notified of tokens needing to be renewed.

**Comment by mpadge** [comment]
This gained a boost in importance this week, with the discovery that GitHub must have somehow changed their token handling. Up until now, the main token shared between `pkgcheck` and `pkgcheck-action` (called "RRT_TOKEN") was a simple repo-level token that could be created by anyone with admin access, yet *did not need to be a personal token*. That recently stopped working, and the use of the token here:
https://github.com/ropensci-review-tools/pkgcheck/blob/86bb65490969a76bf4bfd1ca92a7247ca9d27768/.github/workflows/docker.yaml#L50-L54
now only works with a *personal* token copied over to the equivalent token values here and in `pkgcheck-action`. I'll update the docs straight away to indicate this. The process will then definitely be easier and more robust if we had an account for token creation.

**Comment by maelle** [comment]
what sort of notifications would be relevant? the token expiration?

**Comment by mpadge** [comment]
Yes. Technically they'd just have to watch whatever issue was given here:
https://github.com/ropensci-review-tools/pkgcheck/blob/0728387cf316b51129aea3f73f156eb620144dc7/.github/workflows/monthly.yaml#L26
which is where the token expiration notifications come. Shall we set that up then?

**Comment by mpadge** [comment]
Description of tokens added in [this `ropensci-review-tools` commit](https://github.com/ropensci-review-tools/ropensci-review-tools/commit/fd7e6960db151423b55168cfd116c6ab44ff181e), including a section on "*Assigning 'RRT_TOKEN' updates to a different person*." Maybe that's actually sufficient for now? What do you think @maelle @assignUser :question: 

Rendered section on tokens is [here](https://ropensci-review-tools.readthedocs.io/en/latest/maintenance/tokens.html), with description of maintance-handover [here](https://ropensci-review-tools.readthedocs.io/en/latest/maintenance/tokens.html#assigning-rrt-token-updates-to-a-different-person).
